src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```
@@ -92,24 +157,32 @@
       prev?.value === newValue.value ? prev : newValue
     )
     setSelectedSubject(null)
-  }, [])
+
+    setUnsavedChanges(true)
+    setNeedConfirmation(true)
+  }, 
+    [setUnsavedChanges, setNeedConfirmation])
+
 
   const handleSubjectChange = useCallback((newValue) => {
```

ESLint

[error] Insert `⏎····`

(prettier/prettier)